debian/postinst

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
-
-gtk-update-icon-cache -f '/usr/share/icons/elementary/'
```

Comment:
Do we not need this anymore? What triggers updating the icon-cache then?

Reply:
The following line handles this instead:

https://github.com/pantheon-tweaks/pantheon-tweaks/blob/465a9de7a9c3cb9cb28a854ed9de4d85929aeae0/meson/post_install.py#L9

Also, the path in `debian/postinst` is old actually and [the icons are installed to `hicolor` directory](https://github.com/pantheon-tweaks/pantheon-tweaks/blob/465a9de7a9c3cb9cb28a854ed9de4d85929aeae0/icon/meson.build#L4), so this does nothing.